### ISSUE TITLE
Clarify the instance groups usage

### ISSUE THREAD

**Issue by AndrewDryga** [CloudFire-LLC/cloudfire#2371]
- [x] #2473
- [ ] Explain that all instances within the group should have the same network environments (all resources available) and ideally be the exact replicas of the same VM (eg. part of the same "Instance Group" on Google Cloud, "Auto Scaling Groups" on AWS, Replica/Daemon/Stateful Set in Kubernetes).
- [ ] In the place where we show docker command we should explain that it won't be shown again
- [ ] Add a "Create similar" feature to an instance group, which will open a creating page with all the resource connections copied from an original one
 - [x] #2138

**Comment by jamilbk** [comment]
@AndrewDryga I thought about this more, and I don't think we can rely on "cloning and instance group" to get around the "token not shown again" problem. There are a couple subtle usability issues with that approach:

- Instance groups will be tiered (or usage-based), so the user may already be their max number for their tier
- There's a subtle race condition in which another admin could modify the original Instance group settings between the `Clone` and `Create` actions

Let's hold off on building that for now.

For `ApiToken` in 0.7 we used `Phoenix.encode_and_sign/4` to sign the `api_token.id` plus a `nonce` so that we could generate as many re-usable tokens as we needed to. That design was based on user feedback and worked pretty well. I meant to mention that in standup yesterday but didn't communicate it very clearly.

I think something similar might work for this too, except we store the signing secret per account instead of globally like the cookie and live view secrets. Then upgrading to using X.509 PKI to do the signing / verification later on would be a bit easier too.

For now to avoid having to rip this apart, we can go with "clarifying in the UI" solution. Just wanted to note the above for future reference.

Refs #2138 
Refs #1770
